.github/workflows/static_analysis.yaml

```diff
@@ -145,11 +146,29 @@ jobs:
             echo "Error - CMake version $CURRENT_CMAKE_VERSION does not match the pattern VERSION dd.dd.dd"
             exit 1
           fi
-
-          if [ "$CURRENT_CMAKE_VERSION" == "${EXPECTED_CMAKE_VERSION}" ] || [ "$CURRENT_CMAKE_VERSION" \< "${EXPECTED_CMAKE_VERSION}" ]
```

Comment:
The previous version was fine, no? Did you try simply changing from `<` to `-lt` and removing the quotes?

Reply:
${EXPECTED_CMAKE_VERSION} variable contained string "VERSION 3.8.0" so I decided compare strings.

Reply:
So maybe not have `VERSION` in there?

Reply:
I checked and found that when I using numbers and not strings with `-lt` an output is correct.
I will check how to cast string to float...